<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HalconAlgoTestClass</class>
 <widget class="QMainWindow" name="HalconAlgoTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>829</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HalconAlgoTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnClicked">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>50</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>点击分析图片</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editFilePath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>791</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>E:\practice\halcon\Qt\temp\temp\NGImg\</string>
    </property>
   </widget>
   <widget class="QWidget" name="halconWindow" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>571</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="text">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>110</y>
      <width>341</width>
      <height>651</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>571</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string>Image</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>944</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="HalconAlgoTest.qrc"/>
 </resources>
 <connections/>
</ui>
